apps/managedidentity/managedidentity_test.go

```diff
@@ -221,102 +209,12 @@ func TestCreatingIMDSClient(t *testing.T) {
 				return
 			}
 			if err != nil {
-				t.Fatal("client New() error while creating client")
+				t.Fatal(err)
 			} else {
 				if client.miType.value() != tt.id.value() {
 					t.Fatal("client New() did not assign a correct value to type.")
 				}
 			}
 		})
 	}
-
-}
-func TestCreateIMDSAuthRequest(t *testing.T) {
```

Comment:
I suggested merging this into another test, not simply deleting it; the coverage is important

Reply:
I added that test to `Test_SystemAssigned_Returns_Token_Success`